### PR TITLE
[zh] Update operator.md

### PR DESCRIPTION
**Pull request by ptux**
- fix subject

related upstream PR:
https://github.com/kubernetes/website/pull/30464

### CODE REVIEW

File: content/zh/docs/concepts/extend-kubernetes/operator.md [hunk 1]
```diff
@@ -65,8 +65,8 @@ a [Custom Resource](/docs/concepts/extend-kubernetes/api-extension/custom-resour
 Kubernetes 为自动化而生。无需任何修改，你即可以从 Kubernetes 核心中获得许多内置的自动化功能。
 你可以使用 Kubernetes 自动化部署和运行工作负载， *甚至* 可以自动化 Kubernetes 自身。
 
-Kubernetes {{< glossary_tooltip text="控制器" term_id="controller" >}}
-使你无需修改 Kubernetes 自身的代码，即可以扩展集群的行为。
+Kubernetes 的 {{< glossary_tooltip text="operator pattern" term_id="operator-pattern" >}} 概念
```
Comment (tengqm) [inline content/zh/docs/concepts/extend-kubernetes/operator.md:68]:
```suggestion
Kubernetes 的 {{< glossary_tooltip text="Operator 模式" term_id="operator-pattern" >}}概念
```

**Comment by ptux** [comment]
@tengqm 

Thanks for the code suggestion.

**Pull request merged**